influxdb/tests/server_tests/client_test_with_server.py

```diff
@@ -305,6 +358,46 @@ def test_write_points_DF(self):
             )
         )
 
+    def test_write_points_mixed_type(self):
+        """Test writing points with mixed type and reading them in again."""
+        self.assertIs(True,
+                      self.cli.write_points(mixed_series_dummy_points_part_1))
+        self.assertIs(True,
+                      self.cli.write_points(mixed_series_dummy_points_part_2))
+
+        client_1 = InfluxDBClient('localhost', self.influxd_inst.http_port,
+                                  'root', '', database='db', use_msgpack=False)
```

Comment:
I don't think this test should be added. There is a bug in influxdb (https://github.com/influxdata/influxdb/issues/8707) that causes the JSON response to be parsed as the wrong datatype.

Reply:
Like I said in the description : 

> The test test_write_points_mixed_type fails on purpose now because of a difference between the json and msgpack result returned by the server. 

I can remove the defaulting to json and just make sure all tests run with both json and msgpack client.

I think users would want to be made aware of the consequences of defaulting to msgpack.

Reply:
The consequences you are talking about is a bug that is now fixed with floats being misinterpreted as integers. I agree with you that this bugfix should be mentioned is the change log. 

In any case, you don't want to have `test_write_points_mixed_type` run with the json client, because it fails due to a bug in influxdb, not in this library.